deploy/examples/operator-openshift.yaml

```diff
@@ -125,7 +125,7 @@ data:
   # Set to true to enable host networking for CSI CephFS and RBD nodeplugins. This may be necessary
   # in some network configurations where the SDN does not provide access to an external cluster or
   # there is significant drop in read/write performance.
-  # CSI_ENABLE_HOST_NETWORK: "true"
+  CSI_ENABLE_HOST_NETWORK: "true"
```

Comment:
I see the changes are necessary in the helm chart since the value could never be set to false, but if the default is always set in the operator.yaml, that prevents the [default in the code](https://github.com/rook/rook/blob/release-1.9/pkg/operator/ceph/csi/csi.go#L87) from being used. The code default is preferred so during upgrades we can control the default if it changes. So let's revert these changes in the two operator.yaml files.

Reply:
I understand, done